cspatients/util.py

```diff
@@ -172,7 +185,7 @@ def generate_password_reset_url(request, user):
 
 
 def clean_and_split_string(str):
-    return [x.strip() for x in str.split(",") if x]
+    return [x.strip() for x in str.strip().split(",") if x]
```

Comment:
We should also void using `str` I think that's the representation of the string type in python

Reply:
oh snap, yes that will need to get fixed. thanks